File: CSharp/Algorithm/Sum.cs
using static System.Console;

public class Program {
	public static void Main() {
		var soma = 0;
        foreach (var i in { 3, 7, -6, 10, -1, 0, -1, 4}) if (i > 0) soma++;
        WriteLine("Soma: " + soma);
	}
}

//https://pt.stackoverflow.com/q/413495/101

File: Python/Algorithm/Sum.py
while True:
    numero = int(input('Digite um número inteiro e positivo: '))
    if numero >= 0:
        break
    print('O número não pode ser negativo.')
divisor = 1
soma = 0
for divisor in range(divisor, numero):
    if numero % divisor == 0:
    	soma += divisor
print(soma)

#https://pt.stackoverflow.com/q/482634/101
